packages/patternfly-4/react-core/src/components/Select/SelectMenu.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Select/select';
+import { default as formStyles } from '@patternfly/react-styles/css/components/Form/form';
+import { css } from '@patternfly/react-styles';
+import { SelectConsumer, SelectVariant } from './selectConstants';
+
+// tslint:disable-next-line
+const FocusTrap: any = require('focus-trap-react');
+
+export interface SelectMenuProps {
```

Comment:
missing `extends React.HTMLProps<HTMLElement>`.

Reply:
This errors out due to the `checked` param needing a string[] and not a boolean, and the rendered content can be either a `div` or a `ul`. Is there another extension that would fit better, or do I omit `checked`?

Reply:
You can try to omit checked. It should definitely extend some sort of Element, but it may take some research and omitting params to figure out which one is appropriate in this scenario.

Reply:
I was able to use the extension with Omit typeUtil helper.